internal/output/options.go

```diff
@@ -34,36 +36,47 @@ type WebhookOptions struct {
 	Probe       string        // Server probe behavior.
 }
 
+// GCPPubsubOptions holds configuration for the Google Cloud Pub/Sub output.
 type GCPPubsubOptions struct {
 	Project      string // Project name.
 	Topic        string // Topic name. Will create it if not exists.
 	Subscription string // Subscription name. Will create it if not exists.
 	Clear        bool   // Clear will clear all topics and subscriptions before running.
 }
 
+// KafkaOptions holds configuration for the Kafka output.
 type KafkaOptions struct {
-	Topic string // Topic name. Will create it if not exists.
+	Topic string // Topic is the Kafka topic name. It will be created if it does not exist.
```

Comment:
"Topic is the Kafka topic name. The topic will be created if it does not exist."

("It" is `Topic string` on first reading)

Similar below.